### ISSUE TITLE
Template dependency-monkey parameter THOTH_ADVISER_REQUIREMENTS is required

### ISSUE THREAD

**Issue by harshad16** [thoth-station/thoth-application#1005]
**Describe the bug**
Template dependency-monkey parameter THOTH_ADVISER_REQUIREMENTS is required
however, it is not being satisfied while trigger from management-api

Related-to: https://github.com/thoth-station/management-api/issues/727

logs: 
```
{
  \"kind\":\"Status\",
\"apiVersion\":\"v1\",
\"metadata\":{},
\"status\":\"Failure\",
\"message\":\"Template.template.openshift.io \\\"dependency-monkey\\\" is invalid: template.parameters[2]: Required value: template.parameters[2]: parameter THOTH_ADVISER_REQUIREMENTS is required and must be specified\",
\"reason\":\"Invalid\",
\"details\":{\"name\":\"dependency-monkey\",
\"group\":\"template.openshift.io\",
\"kind\":\"Template\",
\"causes\":[{\"reason\":\"FieldValueRequired\",
\"message\":\"Required value: template.parameters[2]: parameter THOTH_ADVISER_REQUIREMENTS is required and must be specified\",
\"field\":\"template.parameters[2]\"}]},
\"code\":422} "
}
```

**Comment by fridex** [comment]
We should probably escalate https://github.com/thoth-station/thoth-application/pull/982  to unblock Tomas.

**Comment by harshad16** [comment]
This was fixed with #982 
Thanks @fridex
